extensions/amp-ngage-livechat/0.1/amp-ngage-livechat.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS-IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {CSS} from '../../../build/amp-ngage-livechat-0.1.css';
+
+class AmpNgageLiveChat extends AMP.BaseElement {
+  /** @override */
+  buildCallback() {
+    const ja = document.createElement('input');
+    ja.id = 'ngageMobileInvite';
+    ja.type = 'checkbox';
+    this.element.appendChild(ja);
+    const x = document.createElement('label');
+    x.setAttribute('for','ngageMobileInvite');
```

Comment:
We have a [helper function](https://github.com/ampproject/amphtml/blob/master/src/dom.js#L152-L162) that will make a lot of this easier to read.